Tests/js/views/assets/ez-contenteditview-tests.js

```diff
@@ -569,6 +583,17 @@ YUI.add('ez-contenteditview-tests', function (Y) {
             this._testAddDataActionEvent('publishAction');
         },
 
+        "Should set the server side errors to the form view when calling the serverSideErrorCallback": function () {
+            this.view.on('*:publishAction', Y.bind(function (e) {
+                e.serverSideErrorCallback([]);
```

Comment:
you never check that the form edit view receives the right parameters (this exact array)